test/types/sort.test-d.ts

```diff
@@ -0,0 +1,95 @@
+import { expectError } from 'tsd';
+
+import {
```

Comment:
Do we have any tests that test with non-const sort definitions?

Reply:
added some at the top here